src/news.js

```diff
@@ -0,0 +1,50 @@
+class News {
```

Comment:
It feels like you could decompose this into more classes  -- one class I could imagine would be an `Article` class which would know how  to create HTML to display a single article but leaves the fetching of articles to a different part of your code.